client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -201,6 +238,44 @@ describe('FeedbackGeneratorService', function() {
         'ZeroDivisionError: integer division or modulo by zero on line 5');
     });
 
+    it('should throw an error if the line number index is less than 0',
+        function() {
```

Comment:
Style nit: Keep the indentation style uniform with how you did this in the other parts of this file. Looks like the 3 tests here use a 4 space indent for the second line but elsewhere it's 2? So either change all of them to 2 or to 4.

I suppose it's a bit difficult given that this isn't actually uniform throughout the rest of the tests. It's not a big deal but I would say it helps to at least keep all the tests you've written in the same style!